pkg/framework/plugins/defaultevictor/types.go

```diff
@@ -33,4 +33,5 @@ type DefaultEvictorArgs struct {
 	LabelSelector           *metav1.LabelSelector  `json:"labelSelector"`
 	PriorityThreshold       *api.PriorityThreshold `json:"priorityThreshold"`
 	NodeFit                 bool                   `json:"nodeFit"`
+	MinReplicas             uint                   `json:"minReplicas"`
```

Comment:
Please feel free to take over this PR.

At the moment, I have other higher priority items and have low confidence in any movement on this PR (been around since October of last year).

Reply:
I will put it on my todo list where it will probably stay for some time before I find some free cycle for some more comprehensive though. Thank you for pushing this forward.

Reply:
0 replicas mean a pod does not exist. Which is never a case. Fine to keep it as default = disabled.

Reply:
I added a warning message in validator to print that it's a no-op when set to 1